lesson-20/index.js

```diff
@@ -0,0 +1,102 @@
+"use strict"
+
+// task1
+
+// const colors = ['red', 'green', 'blue'];
+
+// console.log(colors.length);
+
+
+// task2
+
+// let animals = ['monkey', 'dog', 'cat'];
+
+// console.log(animals[animals.length - 1]);
+
+
+// task3
+
+// const numbers = [5, 43, 63, 23, 90];
+
+//1
+// numbers.length = 0;
+// console.log(numbers);
+
+//2
+// console.log(numbers.slice(0,0));
+
+
+// task4
+
+// const students = ['Polina', 'Dasha', 'Masha'];
+
+// students.pop();
+// students.push(`Borya`);
+
+
+// students.shift();
+// students.unshift(`Andrey`);
+// console.log(students);
+
+
+// task5
+
+// const cats = ['Gachito', 'Tom', 'Batman'];
+
+//     for( let i = 0; i < cats.length; i += 1){
```

Comment:
```suggestion
//     for(let i = 0; i < cats.length; i += 1) {
```
![image](https://user-images.githubusercontent.com/32203047/159563926-b7d2ce06-64ca-4dce-a707-dd36dcf36b31.png)